src/components/Slides.vue

```diff
@@ -145,6 +154,8 @@ export default {
   methods: {
     searchForSlide () {
       console.log('Pressed search')
+    },
+    cButtonClick () {
```

Comment:
you can add a `console.log()` here or an `alert()`, to allow for testability

Reply:
fixed!